worker_binder.go

```
@@ -186,28 +187,28 @@ func (qs *workerBinder[T, R]) handleQueueSubscription(action string, _ []byte) {
 // It returns a Queue interface that can be used to add jobs to the queue
 func (qs *workerBinder[T, R]) BindQueue() Queue[T, R] {
-	return qs.BindWithQueue(queues.NewQueue[iJob[T, R]]())
+	return qs.WithQueue(queues.NewQueue[iJob[T, R]]())
 }
 
-// BindWithQueue binds an existing queue implementation to the worker
+// WithQueue binds an existing queue implementation to the worker
 // It starts the worker and returns a Queue interface to interact with the queue
-func (qs *workerBinder[T, R]) BindWithQueue(q IQueue) Queue[T, R] {
+func (qs *workerBinder[T, R]) WithQueue(q IQueue) Queue[T, R] {
 	qs.worker.start()
```

golangci-lint (1.64.8)

Error return value of `qs.worker.start` is not checked

(errcheck)